sdk/search/Azure.Search/src/SearchFilter.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Text;
+
+namespace Azure.Search
+{
+    /// <summary>
+    /// The SearchFilter class is used to help construct valid OData filter
+    /// expressions, like the kind used by <see cref="SearchOptions.Filter"/>,
+    /// by automatically replacing, quoting, and escaping interpolated
+    /// parameters.
+    /// <see href="https://docs.microsoft.com/en-us/azure/search/search-filters"/>
+    /// </summary>
+    public static class SearchFilter
+    {
+        /// <summary>
+        /// Create an OData filter expression from an interpolated string.  The
+        /// interpolated values will be quoted and escaped as necessary.
+        /// </summary>
+        /// <param name="filter">An interpolated filter string.</param>
+        /// <returns>A valid OData filter expression.</returns>
+        public static string Create(FormattableString filter) =>
```

Comment:
Why not constructors for an otherwise non-static class?

Reply:
I'm not following what you mean by otherwise non-static?   This is just a static class with a single helper method in it make constructing filters a little easier.

Reply:
Sorry, wasn't considering it was just returning a string. Eyes start to go cross-eyed after a while in large PRs (especially when GitHub tries to load all file diffs in a single page).